app/controllers/all/export/grant_management_and_finance_unit/transfers/projects_controller.rb

```diff
@@ -0,0 +1,42 @@
+class All::Export::GrantManagementAndFinanceUnit::Transfers::ProjectsController < ApplicationController
+  def index
+    authorize Project, :index?
+
+    months = []
+    export_months.each do |month|
+      months << {
+        month: month,
+        count: ProjectsForExportService.new.grant_management_and_finance_unit_transfer_projects(month: month.month, year: month.year).count
+      }
+    end
+    @data = months
+  end
+
+  def show
+    authorize Project, :show?
+    @month = Date.parse("#{year}-#{month}-1")
+  end
+
+  def csv
+    authorize Project, :index?
+
+    projects = ProjectsForExportService.new.grant_management_and_finance_unit_transfer_projects(month: month, year: year)
+    csv = Export::Transfers::GrantManagementAndFinanceUnitCsvExportService.new(projects).call
+
+    send_data csv, filename: "#{year}-#{month}_grant_management_and_finance_unit_transfers_export.csv", type: :csv, disposition: "attachment"
+  end
+
+  private def month
+    params[:month]
+  end
+
+  private def year
+    params[:year]
+  end
+
+  private def export_months
+    12.times.map do |index|
+      Date.today.at_beginning_of_month - index.months
```

Comment:
This might be fine, but I'd be a bit paranoid about the month maths with leap years and februaries; I wonder if it's worth adding a parameter (which could default to `Date.today`) and a few tests around weird dates, to make sure it gets it right every time?